reference/v8.json

```diff
@@ -43,6 +43,37 @@
       "doc": "Default pitch, in degrees. Zero is perpendicular to the surface, for a look straight down at the map, while a greater value like 60 looks ahead towards the horizon. The style pitch will be used only if the map has not been positioned by other means (e.g. map options or user interaction).",
       "example": 50
     },
+    "light": {
+      "anchor": {
+        "type": "enum",
+        "default": "viewport",
+        "values": [
+          "map",
+          "viewport"
+        ],
+        "doc": "Whether extruded geometries are lit relative to map or viewport. The light anchor property will only be used if it has not been set by map options upon initialization.",
+        "example": "map"
+      },
+      "direction": {
+        "type": "array",
+        "value": "number",
+        "default": [1.15, 210, 30],
+        "doc": "Light direction for lighting extruded geometries, in [r radial coordinate, a azimuthal angle, p polar angle] where r indicates the distance from the center of an object to its light, a indicates the position of the light relative to 0º (like a clock), and p indicates the height of the light (from 0º, directly above, to 180º, directly below). The light direction property will only be used if it has not been set by map options upon initialization.",
+        "example": [1.5, 90, 80]
```

Comment:
Maybe more self-documenting, but I hesitate because:
* the naming of spherical coordinates is [so fraught as it is](http://mathworld.wolfram.com/SphericalCoordinates.html), and I'd rather loosely suggest names in the spec docstring than encode a certain naming convention in every style, and
* (related) using an array kind of allows a user to forget their "names" — even in my mind already, arr[0] is the length, arr[1] is the clock angle, arr[2] is the light height. Having to use "radial," "azimuthal," and "polar" in the style or, worse, in a `setLight()` call, seems worse than supplying an array to "direction"